### ISSUE TITLE
Ckan Internatiolization

### ISSUE THREAD

**Issue by yank07** [datagovuk/dgu-vagrant-puppet#39]
How can a use CKAN in another language? I've tried putting

ckan.locale_default= es

I have the following error

File '/vagrant/src/ckanext-dgu/ckanext/dgu/theme/templates/layout_base.html', line 127 in <Expression u"if_(h.check_access('package_create'),'with-publisher','without-publisher')">
  <div id="blackbar" class="${if_(h.check_access('package_create'),'with-publisher','without-publisher')}">
File '/vagrant/src/ckan/ckan/lib/helpers.py', line 667 in check_access
  logic.check_access(action, context, data_dict)
File '/vagrant/src/ckan/ckan/logic/__init__.py', line 296 in check_access
  log.debug('check access NO - %s user=%s "%s"', action, user, str(e))
File '/vagrant/src/ckan/ckan/logic/__init__.py', line 39 in __str__
  return ' - '.join([str(err_msg) for err_msg in err_msgs if err_msg])
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe1' in position 30: ordinal not in range(128)

**Comment by VladimirZD** [comment]
Yank we did a bunch of localizations in our project.. There are 4 solutions (we used them all)
1. CKAN - standard localization (least usefull)
2. Editing ckan templates (strings are harcoded so we changed them to croatian....)
3. Editing drupal templates, We made changes in code and trying to use t(string) funciton, so we can localize that through drupal translation interface
4. Standard drupal translation interface

Mostly we have used options 2 and 3 and 4 ofc... Because croatian is using strange characters "š","Č".. and so on we had to change file encoding on edited files.. They were ANSI and we converted them to UTF-8 and all is working fine..
